### PR TITLE
Andrelouis.NokiaS30MidiSim version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102951)

### CODE REVIEW

**Comment by upintheairsheep** [comment]
portable app

File: manifests/a/Andrelouis/NokiaS30MidiSim/1.0/Andrelouis.NokiaS30MidiSim.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Andrelouis.NokiaS30MidiSim
+PackageVersion: 1.0
+Installers:
+- InstallerUrl: http://onj3.andrelouis.com/phonetones/Software%20and%20Soundbanks/Software/Nokia/Nokia%20Series%2030%20MIDI%20simulator.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (stephengillie) [inline manifests/a/Andrelouis/NokiaS30MidiSim/1.0/Andrelouis.NokiaS30MidiSim.installer.yaml:10]:
```suggestion
  NestedInstallerType: portable
```

**Comment by stephengillie** [comment]
This package installed normally, but errors when launched: 
> Faulting application name: Simulator-050518-ADPCM_support.exe, version: 1.0.0.1, time stamp: 0x428aeafc 
> Faulting module name: Simulator-050518-ADPCM_support.exe, version: 1.0.0.1, time stamp: 0x428aeafc 
> Exception code: 0xc0000005 Fault offset: 0x00008f24 
> Faulting process id: 0x0x2388 
> Faulting application start time: 0x0x1D97C6128661D22 
> Faulting application path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Andrelouis.NokiaS30MidiSim__DefaultSource\Simulator-050518-ADPCM_support.exe 
> Faulting module path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Andrelouis.NokiaS30MidiSim__DefaultSource\Simulator-050518-ADPCM_support.exe 
> Report Id: 5bcd677d-9c83-4b75-8f14-bc4d7b01debf 
> Faulting package full name:  
> Faulting package-relative application ID:  

image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/102951#issuecomment-1530101546

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by ItzLevvie** [comment]
[Policy] Hardware

**Comment by ItzLevvie** [comment]
[Policy] Hardware

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by denelon** [comment]
Supporting custom hardware in validation is "Not Planned".
* https://github.com/microsoft/winget-pkgs/issues/199504

**Pull request closed without merging**